### PR TITLE
Changes to fix game search bug

### PR DESCRIPTION
**Pull request by abindukalpa**
1. Fix Playwright version so that it runs from console
2. Fix to the search since there were changes in placeholder texts

### CODE REVIEW

File: skyVegasTests/utilities/launchGameHelper.ts [hunk 1]
```diff
@@ -8,15 +8,36 @@ export const launchGame = async (
     gameName,
     consoleMessages: string[]
 ) => {
-    const searchForGamesText = 'Search for games...'
-    await expect(page.locator('table.ssc-wldw tbody')).not.toContainText("Loading...");
+    const searchForGamesText = 'SystemSearchSearch for games';
```
Comment (cma06) [inline skyVegasTests/utilities/launchGameHelper.ts]: What is SystemSearchSearch doing?

Comment (abindukalpa) [reply skyVegasTests/utilities/launchGameHelper.ts]: The text box text has changed. It is being used in line 24

Comment (cma06) [reply skyVegasTests/utilities/launchGameHelper.ts]: I can't see the change on my end in NXT?

`<span class="i18n">Search for games...</span>`

Is it a different text for you?

Comment (cma06) [reply skyVegasTests/utilities/launchGameHelper.ts]: So if there were changes in the placeholder text already, this means we should not use it?  Rather use HTML elements like class names / ids as those are less likely to change and would protect us more in the future in my opinion.

I'd not use any text searching and stick to class names/ ids.

Comment (abindukalpa) [reply skyVegasTests/utilities/launchGameHelper.ts]: class names could lead to multiple elements being recognised if the class name is reused for whatever reason. There arent ids being used on this website. 
Also there is no guarantee the class name wont change. Ids would have been the best way, but we dont have them here
The placeholder text is more user centric. If it changes, the tests will need changing.

Comment (cma06) [reply skyVegasTests/utilities/launchGameHelper.ts]: But you don't target one class name. You chain them up so then it shouldn't recognise multiple elements in the DOM...

Comment (cma06) [reply skyVegasTests/utilities/launchGameHelper.ts]: If others in the squad are happy with this solution then we can go ahead. However to me, the fact that the placeholder changed value in less that 24 hours and broke our tests and lead to a code change shows me it should not be used.

Comment (abindukalpa) [reply skyVegasTests/utilities/launchGameHelper.ts]: Per our discussion made the changes :)

**Comment by abindukalpa** [comment]
Using selectors instead of user visible visible locators such as placeholders and text which is the recommended practice. https://playwright.dev/docs/other-locators#css-locator
The UI is not under our control and changes a bit frequently. Css selectors are making the tests more stable.

File: skyVegasTests/utilities/launchGameHelper.ts
Comment (Wilko2001) [inline skyVegasTests/utilities/launchGameHelper.ts:1]: I feel the variables could be given more meaningful names, for example "numbers" variable could be named "balance".

**Review by Wilko2001 — APPROVED** [review]

File: skyVegasTests/utilities/launchGameHelper.ts
Comment (abindukalpa) [reply skyVegasTests/utilities/launchGameHelper.ts:1]: Changed variable names

**Pull request merged**